crates/rolldown_utils/src/hash_placeholder.rs

```diff
@@ -68,7 +68,7 @@ pub fn find_hash_placeholders<'a>(
   results
 }
 
-const CHARS: &[u8] = b"0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_$";
+const CHARS: &[u8] = b"etnriaoscludfpmhg_vybxSCwTEDOkAjMNPFILRzBVHUWGKqJYXZQ$1024368579";
 const BASE: u32 = 64;
 
 pub fn to_base64(mut value: u32) -> String {
```

Comment:
I think we need to split the function into two to avoid changing the filename hash.
```suggestion
pub fn to_base64(value: u32) -> String {
  to_base64_generic::<CHARS>(value)
}

pub fn to_base64_based_on_frequency(value: u32) -> String {
  to_base64_generic::<FREQUENT_CHARS>(value)
}

fn to_base64_generic<const CHARS: &[u8]>(mut value: u32) -> String {
```

Reply:
https://github.com/rolldown/rolldown/blob/a3a4228c16289091eccb66451ce47785f7ecadbe/crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs#L475-L476
replace `to_base64` with `to_base64_frequent`  or keep it

Reply:
we should use `to_base64_based_on_frequency` there